src/op_mode/generate_firewall_rule-resequence.py

```diff
@@ -116,9 +116,18 @@ def convert_rule_keys_to_int(config_dict):
         print('Firewall is not configured')
         exit(1)
 
-    #config_dict =  config.get_config_dict('firewall')
     config_dict = config.get_config_dict('firewall')
 
+    # Remove global-options, group and flowtable as they don't need sequencing
+    if 'global-options' in config_dict['firewall']:
```

Comment:
Is the 'firewall' key needed? `get_config_dict('firewall')` should put the firewall config into the root of the dict.

`if 'global-options' in config_dict:`

Reply:
config_dict content:
```
{
    "firewall": {
        "global-options": {
            "all-ping": "enable"
        },
        "group": {
...
```

The ['firewall'] is needed in this case.